src/Console/Command/ReadmeCommand.php

```diff
@@ -213,7 +213,9 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $help = $command->getHelp();
         $help = str_replace('%command.full_name%', 'php-cs-fixer.phar '.$command->getName(), $help);
         $help = str_replace('%command.name%', $command->getName(), $help);
+        $help = preg_replace('#<<(</.*?>)#', '\\\\$1', $help);
```

Comment:
I added it because of [this change](https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2663/files#diff-a8259d3156d95c86bb9eb9f59fa60b4bR434). But actually Symfony already does this replacement when writing output so yes, this change here is probably unnecessary.